README.md

```diff
@@ -18,10 +18,33 @@ In this Quick Start, we will provide individual examples for each Synopsys secur
 
 These workflows will:
 - Validate Scanning platform-related parameters like project and stream
-- Download the Synopsys Bridge and related adapters
 - Run corresponding Synopsys Bridge commands using the specified parameters
 - Synopsys solution functionality is invoked directly by the Synopsys Bridge, and indirectly by the Synopsys Action
 
+# Prerequisites
+
+Before configuring Synopsys Action into your workflow, note the following prerequisites:
+
+**GitHub Runner Setup:**
+
+- Runners are the machines that execute jobs in a GitHub Actions workflow. To use GitHub runners in your project, GitHub Actions must be enabled for a repository/organization settings in order for required workflows to run (Repository Settings → SelectActions → General → Actions permissions)
+- GitHub runner can be Self-hosted or GitHub-hosted. For installing Self-hosted runners refer [Self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners) and for installing GitHub-hosted runners refer [GitHub-hosted Runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners)
+
+**Configure GitHub Secrets:**
+- Sensitive data such as access tokens, user names, passwords and even URLs must be configured using GitHub secrets (GitHub → Project → Settings → Secrets and Variables → Actions).
+
+**Configure GitHub Token:** 
+
+- `github_token` is required as input when running Black Duck Fix PR, Black Duck/Coverity PR Comment. There are 2 different types of tokens that can be passed to `github_token`:
+
+  1. Token can be github specified `secrets.GITHUB_TOKEN` with required workflow read & write permissions(GitHub → Project → Settings → Actions → General → Workflow Permissions). It will be created by GitHub at start of each workflow run.
+  2. If you need a token that requires permissions that aren't available in the `secrets.GITHUB_TOKEN`, you can create a Personal Access Token(PAT) with required scopes (Select Profile Photo → Settings → Developer Settings → Personal access tokens). For more information, refer [ Granting Additional Permissions ](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#granting-additional-permissions) <br/> PAT must have **repo** scope to perform Black Duck Fix PR, Black Duck/Coverity PR Comment.
+
+**Create workflow:**
+
+- Create a new workflow (GitHub → Project → Actions → New Workflow → Setup a workflow yourself) and configure required fields.
```

Comment:
Are we mentioning anywhere that we can select synopsys-action from marketplace on workflow configuration page and see synopsys-action configuration along with it's parameter for help? 
If no, shall we mention that as well in our readme.

Reply:
update before blackduck, coverity and polaris example
we did same in extension